website/views/shared/modals/quest-rewards.jade

```diff
@@ -12,3 +12,8 @@ script(id='partials/options.social.party.quest-rewards.html', type='text/ng-temp
         =env.t('gold')
     tr(ng-if='::quest.drop.unlock()')
       td {{::quest.drop.unlock()}}
+  div(ng-if='_.filter(quest.drop.items, \'onlyOwner\').length > 0')
```

Comment:
Yep, no problem. I'll try to be better about keeping this sort of logic out of the display layer in the future.